src/backend/d3d12/DeviceD3D12.cpp

```diff
@@ -51,7 +51,12 @@ namespace backend { namespace d3d12 {
 
     nxtSwapChainImplementation CreateNativeSwapChainImpl(nxtDevice device, HWND window) {
         Device* backendDevice = reinterpret_cast<Device*>(device);
-        return CreateSwapChainImplementation(new NativeSwapChainImpl(backendDevice, window));
+
+        nxtSwapChainImplementation impl;
+        impl = CreateSwapChainImplementation(new NativeSwapChainImpl(backendDevice, window));
+        impl.textureUsage = NXT_TEXTURE_USAGE_BIT_PRESENT;
```

Comment:
Would it be easier to simply make this bit the default for all backends, and ignore it for the backends which don't require barriers?

Reply:
We might have swapchains that don't present, for example Chromium's compositor that would rather have `NXT_TEXTURE_USAGE_BIT_SAMPLED`.

Reply:
Ah, that makes sense.